pages/_app.js

```diff
@@ -13,6 +14,19 @@ function MyApp({ Component, pageProps }) {
   const [fishLocation, setFishLocation] = useState("");
   const [markerPosition, setMarkerPosition] = useState([]);
 
+  const [fetchedData, setFetchedData] = useState([]);
+  const [latlng, setLatLng] = useState([]);
+
+  async function fetchAPI() {
+    const res = await fetch("/api/formdata");
+    const data = await res.json();
+    setFetchedData(data.data);
+  }
```

Comment:
Your strategy here is to hold a global state with the "fetched" data and then hand it down to every page component. How do you update the state once it has been modified? It is probably a better strategy to have pages fetch their own data, or use server side props on the pages.